src/utils/string.js

```diff
@@ -241,3 +241,7 @@ export function ensureRegex(strOrRegex, exact = true) {
 export function nlToBr(value) {
   return escapeHtml(value || '').replace(/(\r\n|\r|\n)/g, '<br/>\n');
 }
+
+export function stringifyWithWhiteSpace(obj) {
```

Comment:
The function name should mention JSON somewhere.

Reply:
yup